packages/compass-utils/src/growth-experiments.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  useTrackOnChange,
+  type TrackFunction,
+} from '@mongodb-js/compass-telemetry/provider';
+
+export enum TestName {
+  earlyJourneyIndexesGuidance = 'EARLY_JOURNEY_INDEXES_GUIDANCE_20250328',
+}
+
+export const fireExperimentViewed = ({
```

Comment:
Nit: this should follow React patterns and have the name starting with use, this is a hook after all